web-src/src/common/components/history/executions-log-table.vue

```diff
@@ -58,24 +72,57 @@ export default {
         this.ascending = true;
         this.sortColumn = sortKey;
       }
+    },
+
+    searchIconClickHandler() {
+      if (this.isClearSearchButton) {
```

Comment:
I would suggest to check `searchText` instead

Also, if searchText is empty, probably we could focus the field, when the button is clicked

Reply:
Done.

I believe we should focus the field in any case (whether clearing or clicking search button).